src/poetry/console/application.py

```diff
@@ -237,6 +228,52 @@ def _run(self, io: IO) -> int:
 
         return exit_code
 
+    def _option_get_value(self, io: IO, name: str, default: Any) -> Any:
```

Comment:
nitpick: we could probably do `default: T` and `cast()` return to T. It's not worth the work, I'm just mentioning it because I hate seeing `Any` used.

Reply:
I originally had that, but mypy does not like that since default passed to parameter_option is of type Any.